src/products/components/ProductOrganization/ProductOrganization.tsx

```diff
@@ -130,7 +134,14 @@ const ProductOrganization: React.FC<ProductOrganizationProps> = props => {
             <Typography className={classes.label} variant="caption">
               <FormattedMessage defaultMessage="Product Type" />
             </Typography>
-            <Typography>{maybe(() => productType.name, "...")}</Typography>
+            <Link
```

Comment:
For links that have no side effects - always use native browser a tag `href` tags, this provides better accessibility and user experience (you can know open the view in new tab). See `InternalLink` component